packages/ppx_compose/ppx_compose.0.0.4/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "1.2"
+name: "ppx_compose"
+authors: ["Petter A. Urkedal"]
+maintainer: "paurkedal@gmail.com"
+homepage: "https://github.com/paurkedal/ppx_compose/"
+bug-reports: "https://github.com/paurkedal/ppx_compose/issues"
+dev-repo: "https://github.com/paurkedal/ppx_compose.git"
+license: "LGPL-3 with OCaml linking exception"
+
+build: [["jbuilder" "build" "-p" name "-j" jobs]]
+build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
+
+depends: [
+  "jbuilder" {build & >="1.0+beta7"}
+  "topkg" {build}
+  "topkg-jbuilder" {build}
```

Comment:
It's very unlikely that you need topkg or topkg-jbuilder as part of your build. Are you sure those belong here?

Also, it doesn't seem like you're using topkg at all if you aren't using it to make releases (which I'm guessing from your release here)

Reply:
You're right.  I use it during development (for doc and lint) and for creating a distribution, but it's not needed for opam build.  I'll push a fix in a moment.  Thanks!